recipe/meta.yaml

```
@@ -109,7 +106,7 @@ outputs:
         - test -f ${PREFIX}/lib/cmake/adios2/adios2-config.cmake                      # [unix]
         - test -f ${PREFIX}/lib/libadios2_cxx11.so                                    # [linux]
         - otool -L ${PREFIX}/lib/libadios2_cxx11.dylib                                # [osx]
-        - if exist %LIBRARY_PREFIX%\bin\adios2_cxx11.dll (exit 0) else (exit 1)       # [win]
+        - if exist %LIBRARY_PREFIX%\lib\adios2_cxx11.lib (exit 0) else (exit 1)       # [win]
```

Oh, not `.dll`? I think we want to install a shared lib on Windows, too.

---

Looking at the build log, this is still as before
```suggestion
        - if exist %LIBRARY_PREFIX%\bin\adios2_cxx11.dll (exit 0) else (exit 1)       # [win]
```